index.html

```diff
@@ -2260,6 +2260,84 @@ <h3>Role Mapping Table</h3>
           <p><sup>[Note 2]</sup> This specification does not currently contain guidance for when user agents should fire system alert events. Some guidance may be added to the specification at a later date but it will be a recommendation (SHOULD), not a requirement (MUST).</p>
         </div>
       </section>
+
+
+      <section id="mapping_generic_role">
+        <h3>Role Mapping for Generic Host Language Elements</h3>
+        <p>ARIA 1.2 introduced a new <a class="role-reference" href="#generic"><code>generic</code></a> role, intended for host language elements without a specific or consistent user interface semantic meaning.</p>
+        <p>Generic elements are often used by web authors for a variety of custom grouping, stylistic, and functional containers. Examples of such elements include <code>&lt;div&gt;</code> and <code>&lt;span&gt;</code> in HTML, and <code>&lt;g&gt;</code> in SVG. Earlier versions of HTML contained many generic elememts associated with a default style renderings, such as <code>&lt;b&gt;</code>, <code>&lt;i&gt;</code>, and <code>&lt;blink&gt;</code>.</p>
```

Comment:
a typo fix "elememts" and a bit of expanding on the 'earlier versions of html', to indicate that some of these earlier elements are still in the html standard today.

```suggestion
        <p>Generic elements are often used by web authors for a variety of custom grouping, stylistic, and functional containers. Examples of such elements include <code>&lt;div&gt;</code> and <code>&lt;span&gt;</code> in <abbr title="Hypertext Markup Language">HTML</abbr>, and <code>&lt;g&gt;</code> in <abbr title="Scalable Vector Graphics">SVG</abbr>. Earlier versions of HTML contained many generic elements associated with a default style renderings, such as <code>&lt;b&gt;</code>, <code>&lt;center&gt;</code>, and <code>&lt;blink&gt;</code>. While many of these earlier style-based elements have been deprecated, elements such as <code>&lt;b&gt;</code>, <code>&lt;i&gt;</code> and a number of other generic elements representing Text-level semantics remain.</p>
```

Reply:
Not sure I agree with the use of the word "semantics" here. As a comparison, both `<strong>` and `<b>` are presentational, used as styling hook for bold. In at least some APIs, there is zero difference between those elements and `<span style="font-weight: bold;">`

Reply:
@cookiecrook 

>  In at least some APIs, there is zero difference between those elements (strong, b) and font-weight: bold;

Is this not a bug of the API? Because strong and b have a clearly defined semantics according to the HTML specification and the chapter in which this is defined is called not without reason: ["Text-level semantics" ](https://html.spec.whatwg.org/multipage/text-level-semantics.html).

In particular, strong has semantics. Whether b has semantics is debatable.

Reply:
@cookiecrook as JAWS-test mentioned, this is what these elements are classified under in HTML.

Reply:
This is incorporated into the diff now. Marking as resolved.